content/posts/2019-03-11--juben/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: 三校联合剧本杀等你来
+category: "活动"
+cover: cover.jpeg
+link: https://mp.weixin.qq.com/s/nu6PXh4QR1WnUXhrBtVn2A
+author: NUCSSA IT
+---
+三校联合剧本杀等你来!
```

Comment:
solid work 👍  and if we need to put this post on the latest position, we might want to change the date in the file name, since the lastest one is 2020-03-13